__tests__/chat.test.js

```diff
@@ -206,6 +216,18 @@ describe('Chat Methods', () => {
       expect(read1.messages[0].content.type).toEqual('join')
       expect(read1.messages[0].sender.username).toEqual(config.bots.bob1.username)
 
+      let bobMessageCount = 0
+      const bobOnMessage = async message => bobMessageCount++
+      bob.chat.watchChannelForNewMessages(teamChannel, bobOnMessage)
+      bob.chat.watchChannelForNewMessages(generalChannel, bobOnMessage)
+      await timeout(500)
+      await alice1.chat.send(generalChannel, message)
+      await timeout(500)
```

Comment:
Do we wanna check the received message is from the specified channel? Same below. Doesn't seem absolutely necessary though.

Reply:
we do want to bring back concurrency tests. I think the biggest issue with them is that there are some tests that use the same actual channels. For example, I think there's an inline chat send of some money from alice to bob and then alice or bob looks at the most recent message to make sure it has that info. The tests will have to change a bit.